xml/System.Configuration/RsaProtectedConfigurationProvider.xml

```diff
@@ -25,7 +25,7 @@
       <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- The <xref:System.Configuration.RsaProtectedConfigurationProvider> class gives you a way to encrypt sensitive information stored in a configuration file, which helps protect it from unauthorized access. You use the built-in <xref:System.Configuration.RsaProtectedConfigurationProvider> instance by declaring the provider and making appropriate settings in the configuration file instead of creating an instance of this class, as shown in the example later in this topic.  
+ The <xref:System.Configuration.RsaProtectedConfigurationProvider> class gives you a way to encrypt sensitive information stored in a configuration file, which helps protect it from unauthorized access. You use the built-in <xref:System.Configuration.RsaProtectedConfigurationProvider> instance by declaring the provider and making appropriate settings in the configuration file instead of creating an instance of this class, as shown in this topic's examples.
```

Comment:
my suggestion: `as shown in the [Examples](#examples) section.`

Reply:
Thanks for the feedback! I like the idea of using a link.